frontend/src/components/VariantListPage/VariantListVariants.tsx

```diff
@@ -30,7 +30,7 @@ import {
 } from "../../types";
 
 import MultipleSelect from "../MultipleSelect";
-
+import Highlighter from "react-highlight-words";
```

Comment:
**thought**: I think we're going to squash all these commits into a single commit anyways, but just wanted to note that if that wasn't the case, this might be a good place to use interactive rebasing (`git rebase -i origin/main`) to re-order the commits.

Currently in the commit history, this would break, as we're importing this package before its actually added as a dependency. Interactive rebasing is a powerful way to rewrite history to change things like this.

Again - not neccesary here since we're going to squash, just wanted to mention is as a good tool to have!